src/ansible_pygments/lexers.py

```diff
@@ -34,12 +34,14 @@
 
 """Pygments lexers for ansible console output."""
 
-# pylint: disable=consider-using-f-string
+from __future__ import annotations
```

Comment:
I don't see us really needing this boilerplate. Let's not add it unless necessary.

Reply:
66f5816d533a7b3e8b83e7400d4d3ef7387974ed